tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -1773,6 +1787,606 @@ function ( $location ) use ( &$redirect_url ) {
+		try {
+			$this->mock_gateway->process_checkout_session_redirect( $order->get_id() );
+			$this->fail( 'Expected redirect to be triggered' );
+		} catch ( \RuntimeException $e ) {
+			$this->assertSame( 'redirect_intercepted', $e->getMessage() );
+		} finally {
+			remove_all_filters( 'wp_redirect' );
+		}
+
+		$this->assertSame( self::MOCK_RETURN_URL_AFTER_REDIRECT, $captured );
+		$this->assertEquals( OrderStatus::PROCESSING, wc_get_order( $order->get_id() )->get_status() );
+	}
```

PHPMD (2.15.0)

[error] The method test_process_checkout_session_redirect_short_circuits_for_already_processed_order is not named in camelCase. (undefined)

(CamelCaseMethodName)